### PR TITLE
added visual studio code text.rst

### PR DESCRIPTION
**Pull request by josephworks**

### CODE REVIEW

**Comment by ST-DDT** [comment]
Has VSC direct support for reST or do you need a plugin for that? (and add that information to the introduction)

**Review by ST-DDT — APPROVED** [review]

**Comment by phit** [comment]
needs a plugin, e.g. https://marketplace.visualstudio.com/items?itemName=lextudio.restructuredtext

**Comment by josephworks** [comment]
done

**Review by ST-DDT — APPROVED** [review]

File: source/preparing/text.rst [hunk 1]
```diff
@@ -19,6 +19,16 @@ depth about the features of Atom's Git integration. If you're interested in addi
...
 
+Visual Studio Code
+~~~~~~~~~~~~~~~~~~
+
+Visual Studio Code is a source code editor developed by Microsoft for Windows, Linux and macOS. It includes support for debugging, 
+embedded Git control, syntax highlighting, intelligent code completion, snippets, and code refactoring.
+
+**Visual Studio Code** can be downloaded for free from https://code.visualstudio.com/.
+
+reST Extention can be downloaded for free from https://marketplace.visualstudio.com/items?itemName=lextudio.restructuredtext
```
Comment (Inscrutable) [inline source/preparing/text.rst]: Spelling error. Extention -> Extension

File: source/preparing/text.rst [hunk 2]
```diff
@@ -19,6 +19,16 @@ depth about the features of Atom's Git integration. If you're interested in addi
 
 **Atom** can be downloaded for free from https://atom.io/.
 
+Visual Studio Code
+~~~~~~~~~~~~~~~~~~
+
+Visual Studio Code is a source code editor developed by Microsoft for Windows, Linux and macOS. It includes support for debugging,
```
Comment (Inscrutable) [inline source/preparing/text.rst]: Please keep line lengths 120 characters or below, to avoid wrapping. (It's one of our guidelines).

**Review by Inscrutable — CHANGES_REQUESTED** [review]
Thanks for the additions. Can you please fix the quibbles I have highlighted?

**Review by Inscrutable — APPROVED** [review]

**Pull request merged**